VSCodium (1.64.2, undefined, desktop)
Jupyter Extension Version: 2022.1.1301856528.
Python Extension Version: 2022.0.1814523869.
Info 23:18:14: ZMQ install verified.
Error 23:18:14: Jupyter Extension (Error in Failed to update settings.json, method:fixSettingsFile): EntryNotFound (FileSystemError): Unable to read file '/home/justin/.config/Code/User/settings.json' (Error: Unable to resolve nonexistent file '/home/justin/.config/Code/User/settings.json')
    at Function._handleError (/usr/share/codium/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:93:83879)
    at Object.readFile (/usr/share/codium/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:93:82793)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at m.readFile (/home/justin/.vscode-oss/extensions/ms-toolsai.jupyter-2022.1.1301856528/out/client/extension.js:124:42473)
    at g.fixSettingsFile (/home/justin/.vscode-oss/extensions/ms-toolsai.jupyter-2022.1.1301856528/out/client/extension.js:66:522156)
    at async Promise.all (index 0)
    at g.updateSettings (/home/justin/.vscode-oss/extensions/ms-toolsai.jupyter-2022.1.1301856528/out/client/extension.js:66:523906) {
  code: 'FileNotFound'
}
Error 23:18:14: Jupyter Extension (Error in Failed to update settings.json, method:fixSettingsFile): EntryNotFound (FileSystemError): Unable to read file '/home/justin/Documents/Pr/python/vetcalc/.vscode/settings.json' (Error: Unable to resolve nonexistent file '/home/justin/Documents/Pr/python/vetcalc/.vscode/settings.json')
    at Function._handleError (/usr/share/codium/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:93:83879)
    at Object.readFile (/usr/share/codium/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:93:82793)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at m.readFile (/home/justin/.vscode-oss/extensions/ms-toolsai.jupyter-2022.1.1301856528/out/client/extension.js:124:42473)
    at g.fixSettingsFile (/home/justin/.vscode-oss/extensions/ms-toolsai.jupyter-2022.1.1301856528/out/client/extension.js:66:522156)
    at async Promise.all (index 1)
    at g.updateSettings (/home/justin/.vscode-oss/extensions/ms-toolsai.jupyter-2022.1.1301856528/out/client/extension.js:66:523906) {
  code: 'FileNotFound'
}
Info 23:18:14: Experiment status for python is {"enabled":true,"optInto":[],"optOutFrom":[]}
Info 23:18:14: Getting activation commands for /bin/python3 are not cached. May take a while.
Info 23:18:14: Process Execution: > /bin/python3 -m pip list
> /bin/python3 -m pip list
Info 23:18:43: Shutting down notebooks for 45e4e4bd-03b9-4a2e-bd00-a137e11c2847
Info 23:18:43: Disposing HostJupyterExecution 18d00821-c9cd-46f5-abb4-44c5ce676c60
Info 23:18:43: Disposing super HostJupyterExecution 18d00821-c9cd-46f5-abb4-44c5ce676c60
Info 23:18:43: Disposing HostJupyterExecution 18d00821-c9cd-46f5-abb4-44c5ce676c60
Info 23:18:43: Finished disposing HostJupyterExecution  18d00821-c9cd-46f5-abb4-44c5ce676c60
Info 23:18:43: Cleaning up server cache 18d00821-c9cd-46f5-abb4-44c5ce676c60
Info 23:18:43: Finished disposing HostJupyterExecution  18d00821-c9cd-46f5-abb4-44c5ce676c60
